magda-storage-api/src/createApiRouter.ts

```diff
@@ -26,7 +26,23 @@ export default function createApiRouter(options: ApiRouterOptions) {
         return res.status(200).send("OK");
     });
 
-    // Download an object
+    /**
+     * @apiGroup Storage API
+     *
+     * @api {get} /v0/{bucket}/{fieldid} Request to download an object in {bucket} with name {fieldid}
+     *
+     * @apiDescription Downloads an object
+     *
+     * @apiParam (Request body) {string} bucket The name of the bucket under which the requested object is
+     * @apiParam (Request body) {string} fieldid The name of the object being requested
+     *
+     * @apiSuccessExample {text} 200
+     *    Contents of a file
+     *
+     * @apiError {string} status NOT FOUND
+     *
+     * @apiErrorExample {string} No such object with fileId {fileid} in bucket {bucket}
```

Comment:
This is breaking your build.
```suggestion
     * @apiErrorExample {string} 404
     *      No such object with fileId {fileid} in bucket {bucket}
```

You can test without pushing by running `yarn run build` in `magda-apidocs` and running an http server in `magda-apidocs/build` :)